contracts/src/paths.rs

```diff
@@ -0,0 +1,12 @@
+//! Module for common paths used for generated contract bindings.
+
+use std::path::{Path, PathBuf};
+
+/// Path to file containing address of a contract deployed to a test network.
+pub fn contract_address_file(name: &str) -> PathBuf {
```

Comment:
For rinkeby/mainnet, will we also need these `.addr` files or will we get the deployed addresses otherwise?

Reply:
Rinkeby and mainnet addresses are included as part of the artifact JSON files (they get injected from the `networks.json` file). These are only needed for testnet deployments (the reason why we had to first run `./scripts/setup_contracts.sh` before building on Travis).